wnw.py

```diff
@@ -0,0 +1,171 @@
+"""Utilities for the word-nonword study"""
+
+import os.path as op
+
+
+WNW_DEFAULT_ROOT = "/data/NIMH_SFIM/handwerkerd/DenoisingPilot"
```

Comment:
Hardcoded path?

Reply:
This is hardcoded because the project is on a shared biowulf directory for the primary members; if people have a different location, they can override it in the WnwProject constructor (see `wnw.py` L347).